lib/literature.ex

```diff
@@ -490,4 +490,20 @@ defmodule Literature do
   def delete_tag(%Tag{} = tag) do
     Repo.delete(tag)
   end
+
+  @doc """
+  Updates cloudflare.
+  """
+  def update_cloudflare(body \\ %{}) do
+    if Config.cloudflare_config() do
+      "#{Config.cloudflare_api_host()}/zones/#{Config.cloudflare_identifier()}"
```

Comment:
did you think how this works for when a post is updated? Perhaps move caching plug in the router to only affect non-index pages?
```
live("/", BlogLive, :index, route_opts)
live("/tags", BlogLive, :tags, route_opts)
live("/authors", BlogLive, :authors, route_opts)
get("/posts/rss.xml", RSSController, :rss, as: session_name)
```

Reply:
Hi @samhamilton, I used this API https://api.cloudflare.com/#zone-purge-files-by-cache-tags,-host-or-prefix.
since we are creating dynamic paths for the public pages, matching with slug might work.

Reply:
@christianlj thats an enterprise only feature and we only use the pro plan